### ISSUE TITLE
Ring Fit Adventure use too much gpu on windows with RX570

### ISSUE THREAD

**Issue by JasonChen89** [yuzu-emu/yuzu#9901]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Early Access 3412

### Description of Issue

In the Ring Fit Adventure game, when the coach starts demonstrating the first two movements(the coach is big), the GPU usage is extremely high. Especially when the coach demonstrates the first two squats, the GPU usage will reach 100%, and the FPS will be reduced from 60 to about 20. But sometimes yuzu work fine.
When the coach gets smaller, the GPU usage will also decrease, and the FPS will change back to 60.

### Expected Behavior

![low_gpu1](https://user-images.githubusercontent.com/18242607/222949726-b3f43d86-41de-492a-abd8-bd36bc1c3f9b.png)

![low_gpu2](https://user-images.githubusercontent.com/18242607/222949737-ed8f4d91-fc22-4ed9-93a8-885a1edb4201.png)

![low_gpu3](https://user-images.githubusercontent.com/18242607/222949754-47095fe4-138a-47cc-a922-071255cc8bcb.png)

![low_gpu4](https://user-images.githubusercontent.com/18242607/222949931-1de6316b-04db-4381-9837-30214836f138.png)

![low_gpu5](https://user-images.githubusercontent.com/18242607/222950054-eeb0c877-e82b-4f64-9c20-6e7cc45a127b.png)

### Reproduction Steps

![high_cpu1](https://user-images.githubusercontent.com/18242607/222948993-dd778799-e1db-48c8-b7ff-7689fa41cf94.png)

![high_cpu2](https://user-images.githubusercontent.com/18242607/222949587-68489f16-84eb-4a61-b7ca-e56721017a17.png)

![high_cpu3](https://user-images.githubusercontent.com/18242607/222949788-1e28f726-51dc-45cf-bea0-81b35e305be0.png)

![high_cpu4](https://user-images.githubusercontent.com/18242607/222950001-0539af61-f0ef-4cc7-9c45-9556df76e39f.png)

### Log File

log file
[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/10891321/yuzu_log.txt)

### System Configuration

CPU: 5600x
GPU/Driver: XFX RX570 8G / WHQL-AMD-Software-Adrenalin-Edition-23.2.2-Win10-Win11-Feb22
RAM: 32G
OS: windosws 10 (1809)

**Comment by unclesammy** [comment]
最新版也没解决么，我 EA3396,1050ti也有这问题，估计是配置问题。

**Comment by JasonChen89** [comment]
> 最新版也没解决么，我 EA3396,1050ti也有这问题，估计是配置问题。这做瑜伽就很难受了 你试试开下这个有用么‘Decode ASTC textures asynchronously (Hack)’？

After I turned on this feature, the problem also persisted.

**Comment by unclesammy** [comment]
> > 最新版也没解决么，我 EA3396,1050ti也有这问题，估计是配置问题。这做瑜伽就很难受了 你试试开下这个有用么‘Decode ASTC textures asynchronously (Hack)’？
> 
> After I turned on this feature, the problem also persisted.

I think you can solve this by using the latest YUZU,and following this configure
![image](https://user-images.githubusercontent.com/56534652/226082220-fa2f1bbf-dafe-4a27-9316-ddd677ac154e.png)

**Comment by unclesammy** [comment]
> > I think you can solve this by using the latest YUZU,and following this configure ![image](https://user-images.githubusercontent.com/56534652/226082220-fa2f1bbf-dafe-4a27-9316-ddd677ac154e.png)
> 
> I've tried all the settings and the problem is still the same, this shouldn't be a configuration issue. As far as I know, this problem has been present from mainline 999 to the latest version EA 3461.
> 
> You can try the two game levels I showed in the screenshot above, one is bound to have this problem, and the other is not going to appear.

yeah,I think I met the same problem when I'm in the boss fight the frame rate has some significantly drop.

**Comment by unclesammy** [comment]
> > yeah,I think I met the same problem when I'm in the boss fight the frame rate has some significantly drop.
> > And also do u know how to solve the 'left button'(button for checking your personal info and change the intensity) problem?
> 
> I seem to have seen official news that the page that requires an internet connection in the game is not supported for the time being.

I ran into the same problem where I was playing my custom list.When the tips shows in big size frames only retain around 20fps.I try the lowest setting.But it seem for that specific scenario the frame will drop anyway.

**Comment by B1anYu** [comment]
Same question. I suggest you try these settings:
Resolution 540p or 720p
Accuracy Level: Normal
ASTC Recompression Method: BC1
It may gets 50~60 FPS.

**Comment by unclesammy** [comment]
> Same question. I suggest you try these settings: Resolution 540p or 720p Accuracy Level: Normal ASTC Recompression Method: BC1 It may gets 50~60 FPS.
I'm using the latest main version, I find that maybe some tips get a 40~50 fps in your setting.But some still drop like below 30.Overall it still unstable when ran into that scenario.

**Comment by unclesammy** [comment]
> Same question. I suggest you try these settings: Resolution 540p or 720p Accuracy Level: Normal ASTC Recompression Method: BC1 It may gets 50~60 FPS.

I'm using the latest main version, I find that maybe some tips get a 4050 fps in your setting.But some still drop like below 30.Overall it still unstable when ran into that scenario.

**Comment by Maelstrom96** [comment]
Bug is still present on Build 1673.

Had hope that https://github.com/yuzu-emu/yuzu/pull/12543 might've fixed it but doesn't seem like it did. Weirdly, not all videos results in slow framerate, and only the exercise video tips does.

**Comment by EgoBizarro** [comment]
Yeah, I've also always had this issue. It's in specific battle areas and it's the most severe when the tutorial window on the left is bigger (die most exercises it gets smaller after the first two reps). The only thing that helps my FPS quite a bit there (though it's still far from full speed) is switching to handheld mode. This removes the DOF effect, which seems to be rather heavy on the GPU, and maybe some bloom.

**Comment by Itukii** [comment]
Bug is still present on Build 1689.
